### ISSUE TITLE
Allow to customize parameters passed to verify in DafnyServer

### ISSUE THREAD

**Issue by ssaavedra** [DafnyVSCode/Dafny-VSCode#41]
It would be great to be able to configure at the editor's settings.json the parameters that are passed to DafnyServer on verify.

Currently this is implemented on `server/backend/dafny/dafnyServer.js` as:

```js
    sendVerificationRequest(request) {
        if (request.verb === stringRessources_1.DafnyVerbs.CounterExample || request.verb === stringRessources_1.DafnyVerbs.Verify) {
            this.statusbar.changeServerStatus(stringRessources_1.StatusString.Verifying);
        }
        const task = {
            args: [],
            filename: vscode_uri_1.default.parse(request.document.uri).fsPath,
            source: request.source,
            sourceIsFile: false,
        };
```

I'm manually changing on the extension code the `args: []`, in my case to add `args: ['/vcsLoad:1']`, but it would be great if that was exported to be user-configurable.

**Comment by fabianhauser** [comment]
Hi @ssaavedra 

That sounds like a great addition. Would you mind opening a pull request for that? 🙂 

Adding the setting should be possible by extending [the Dafny Settings Interface](https://github.com/DafnyVSCode/Dafny-VSCode/blob/1f612f675d2315e19c46c32f85b501d50efd1bb2/server/src/backend/dafnySettings.ts) and adding it to the [package.json](https://github.com/DafnyVSCode/Dafny-VSCode/blob/develop/client/package.json#L132) of the client part.

**Comment by ssaavedra** [comment]
Thanks for the cue, I just opened #42 :wink: 

I have no idea about VSCode extensions (I've using the editor for a couple of hours, have been using Emacs and still doing so), so please feel free to review any issues.

Also, it seems that configuration changes are not picked up automatically by the extension and that it instead needs to be reloaded (by means of `> Reload window`). I suppose that is a different issue regarding the onDidChangeConfiguration event (from what I has just been reading).

**Comment by ssaavedra** [comment]
Since you just opened #43 which is the only remaining question here (now that #42 is merged), I think this issue can be now closed :)

**Comment by fabianhauser** [comment]
Great, thanks for your contribution! Your changes are included in the newest release (v0.16.0) which should be public in a couple of minutes 😊

I opened a new issue #43 to address the configuration reloading (seems that it was not properly implemented all along...)
